libcontainer/cgroups/systemd/unified_hierarchy.go

```diff
@@ -21,6 +20,8 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+const UnifiedMountpoint = fs2.UnifiedMountpoint
```

Comment:
Can we just directly use `fs2.UnifiedMountpoint` without aliasing?

Reply:
Sure, I just thought it's cleaner that way. Will fix

Reply:
fixed